2023-12-12 15:16:47,131 file.manager INFO 2.0.1
2023-12-12 15:16:47,131 file.manager INFO Copyright (C) 2003-2022, Noriaki Ando and OpenRTM development team,
2023-12-12 15:16:47,131 file.manager INFO   Intelligent Systems Research Institute, AIST,
2023-12-12 15:16:47,147 file.manager INFO Copyright (C) 2022, Noriaki Ando and OpenRTM development team,
2023-12-12 15:16:47,147 file.manager INFO   Industrial Cyber-Physical Research Center, AIST,
2023-12-12 15:16:47,147 file.manager INFO   All right reserved.
2023-12-12 15:16:47,147 file.manager INFO Manager starting.
2023-12-12 15:16:47,147 file.manager INFO Starting local logging.
2023-12-12 15:16:47,147 file.manager.namingmanager INFO NameServer connection succeeded: corba/localhost
2023-12-12 15:16:51,227 file.ManagerServant INFO Master manager not found
2023-12-12 15:16:51,227 file.LocalServiceAdmin INFO All the local services are enabled.
2023-12-12 15:16:51,242 file.manager INFO Component type conf file: sarch_spot.conf loaded.
2023-12-12 15:16:51,242 file.ec_base INFO setRate(1000000.000000) done
2023-12-12 15:16:51,242 file.ec_base INFO setRate(1000.000000) done
2023-12-12 15:16:51,242 file.rtobject INFO 1 execution context was created.
2023-12-12 15:16:51,242 file.rtobject INFO Initial active configuration set is default.
2023-12-12 15:17:05,302 file.ManagerServant INFO Master manager not found
2023-12-12 15:17:07,135 file.SdoServiceAdmin WARNING No available SDO service in the factory: 
2023-12-12 15:17:07,136 file.SdoServiceAdmin ERROR type  not exists.
2023-12-12 15:17:15,340 file.ManagerServant INFO Master manager not found
2023-12-12 15:17:25,403 file.ManagerServant INFO Master manager not found
2023-12-12 15:17:35,420 file.ManagerServant INFO Master manager not found
2023-12-12 15:17:45,468 file.ManagerServant INFO Master manager not found
2023-12-12 15:17:52,064 file.rtobject ERROR Traceback (most recent call last):
  File "C:\Users\reore\AppData\Local\Programs\Python\Python311\Lib\site-packages\OpenRTM_aist\DataFlowComponentBase.py", line 123, in on_execute
    ret = self.onExecute(ec_id)
          ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\reore\workspace\sarch_spot\sarch_spot.py", line 313, in onExecute
    draw_lines_updated(image_array, center_of_image, 15, specific_color, road_colors, ignore_radius_ratio)
  File "C:\Users\reore\workspace\sarch_spot\sarch_spot.py", line 267, in draw_lines_updated
    radius = googlemap_scale_level_list[googlemap_scale_level] * (max_photo_distance  - ignore_radius_ratio) #ê¸ÇÇ«Ç±Ç‹Ç≈à¯Ç≠Ç©
             ~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^
TypeError: list indices must be integers or slices, not str

2023-12-12 15:17:55,512 file.ManagerServant INFO Master manager not found
2023-12-12 15:17:58,627 file.ec_worker ERROR State of the RTC is not ACTIVE_STATE.
